lib/watchlist_edits.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class WatchlistEdits
```

Comment:
Ideally, you want to re-use whatever logic you can from WikiEdits, and just include the unique logic related to watchlists in the new file.